packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/vault/models/AsymmetricStoreRequest.java

```diff
@@ -0,0 +1,77 @@
+package cloud.pangeacyber.pangea.vault.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+public class AsymmetricStoreRequest extends CommonStoreRequest{
+    @JsonProperty("algorithm")
+    AsymmetricAlgorithm algorithm = null;
+
+    @JsonInclude(Include.NON_NULL)
```

Comment:
this can be null

Reply:
Yes, that's why I don't include if it's null. It should take default value in that case